### ISSUE TITLE
panic: runtime error: integer divide by zero

### ISSUE THREAD

**Issue by bhaskar2018** [s3tester/s3tester#18]
Trying to run a mixedworload and it crashes.

```
root@VM109:~/s3tester-v2.1.0-linux-x64# screen -S s3tester ./s3tester -concurrency=5 -size=1048576 -requests=2000 -endpoint=https://xxxxxxxxxx:443 -prefix=mixedworkload -bucket=bhaskar04022020 -workload test.json 
panic: runtime error: integer divide by zero

goroutine 1 [running]:
main.calcStats(0xc000ba76a8, 0x5, 0x1d59de)
        /home/vlad/go/src/github.com/s3tester/s3tester/s3tester.go:460 +0x18b
main.setupResultStat(0xc000ba76a8)
        /home/vlad/go/src/github.com/s3tester/s3tester/s3tester.go:448 +0x49
main.processTestResult(0xc000ba76a8, 0x5, 0x100000, 0xc00007d800, 0x1, 0x1, 0x957efc, 0x3, 0x7fffcd918204, 0xf, ...)
        /home/vlad/go/src/github.com/s3tester/s3tester/s3tester.go:435 +0x74
main.runtest(0x5, 0x100000, 0xc00007d800, 0x1, 0x1, 0x957efc, 0x3, 0x7fffcd918204, 0xf, 0x7fffcd9181ee, ...)
        /home/vlad/go/src/github.com/s3tester/s3tester/s3tester.go:187 +0x1cd
main.main()
        /home/vlad/go/src/github.com/s3tester/s3tester/s3tester.go:576 +0x90d 
```

```
root@VM109:~/s3tester-v2.1.0-linux-x64# cat test.json 
{'mixedWorkload':[{'operation':'put','ratio':25},{'operationType':'get','ratio':25},{'operationType':'updatemeta','ratio':25},{'operationType':'delete','ratio':25}]}
```

Not sure what am I missing.

**Comment by sali619** [comment]
Same above issues observed when try running mixedworkloadprofile:
[root@eosnode-1 ~]# go run s3tester -bucket test1 -concurrency=10 -size=50 -requests=50 -endpoint="https://s3.seagate.com" -prefix=3 -workload=workload.json
panic: runtime error: integer divide by zero

goroutine 1 [running]:
main.calcStats(0xc0000f55b0, 0xa, 0xcfb72f)
        /root/go/src/s3tester/s3tester.go:460 +0x188
main.setupResultStat(0xc0000f55b0)
        /root/go/src/s3tester/s3tester.go:448 +0x49
main.processTestResult(0xc0000f55b0, 0xa, 0x32, 0xc0000819e0, 0x1, 0x1, 0x9bb445, 0x3, 0x7fff1b109767, 0x5, ...)
        /root/go/src/s3tester/s3tester.go:435 +0x74
main.runtest(0xa, 0x32, 0xc0000819e0, 0x1, 0x1, 0x9bb445, 0x3, 0x7fff1b109767, 0x5, 0x7fff1b1097bc, ...)
        /root/go/src/s3tester/s3tester.go:187 +0x1cd
main.main()
        /root/go/src/s3tester/s3tester.go:576 +0x925
exit status 2
[root@eosnode-1 ~]#

**Comment by vmore** [comment]
This was due to an error int he workload file. The example in the readme was fixed by #23 . The same problem is discussed here and solved by changing the first occurrence of "operation" to "operationType". See #24
